WordPress/src/main/java/org/wordpress/android/ui/reader/services/discover/ReaderDiscoverServiceStarter.kt

```diff
@@ -22,30 +19,22 @@ object ReaderDiscoverServiceStarter {
     const val ARG_DISCOVER_TASK = "discover_task"
 
     fun startService(context: Context, task: DiscoverTasks): Boolean {
-        if (VERSION.SDK_INT < VERSION_CODES.O) {
-            val intent = Intent(context, ReaderDiscoverService::class.java)
-            intent.putExtra(ARG_DISCOVER_TASK, task)
-            context.startService(intent)
+        val componentName = ComponentName(context, ReaderDiscoverJobService::class.java)
+        val extras = PersistableBundle()
+        extras.putInt(ARG_DISCOVER_TASK, task.ordinal)
+        val jobInfo = Builder(JobServiceId.JOB_READER_DISCOVER_SERVICE_ID, componentName)
+            .setRequiresCharging(false)
+            .setRequiredNetworkType(JobInfo.NETWORK_TYPE_ANY)
+            .setOverrideDeadline(0) // if possible, try to run right away
+            .setExtras(extras)
+            .build()
+        val jobScheduler = context.getSystemService(Context.JOB_SCHEDULER_SERVICE) as JobScheduler
+        val resultCode = jobScheduler.schedule(jobInfo)
+        if (resultCode == JobScheduler.RESULT_SUCCESS) {
+            AppLog.i(READER, "reader discover service > job scheduled")
         } else {
-            // schedule the JobService here for API >= 26. The JobScheduler is available since API 21, but
-            // it's preferable to use it only since enforcement in API 26 to not break any old behavior
```

Comment:
Minor (🔍): On all other such changes you made, chose to keep a similar comment. Maybe we should keep this comment as well? 😊

Reply:
Good point - actually, I decided to go with removing this comment in other places as this is explanation why we don't use JobScheduler in API 21+, even though it was already available. And this code is now removed.